napari/utils/colormaps/colormap_utils.py

```diff
@@ -358,7 +368,17 @@ def _color_random(n, *, colorspace='lab', tolerance=0.0, seed=0.5):
         elif colorspace == 'rgb':
             raw_rgb = random
         else:  # 'lab' by default
-            raw_rgb = colorconv.lab2rgb(random * LABRNG + LABMIN)
+            # The values in random are in [0, 1], but since the LAB colorspace
+            # is not exactly contained in the unit-box, some 3-tuples might not
+            # be valid LAB color coordinates. scikit-image handles this by projecting
+            # such coordinates into the colorspace, but will also warn when doing this.
+            with warnings.catch_warnings():
```

Comment:
I think the same problem (i.e. invalid coordinates) could occur for the LUV colorspace, but the corresponding scikit-image function does not seem to warn in that case, so I left it alone for now.